### PR TITLE
ufw: add more information link

### PR DESCRIPTION
**Pull request by CleanMachine1**
For #6062

### CODE REVIEW

**Review by bl-ue — APPROVED** [review]

**Review by mfrw — APPROVED** [review]
LGTM

**Comment by CleanMachine1** [comment]
Sorry this was such a small and stupid, change I just wanted to do 1 before sleep :)

File: pages/linux/ufw.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 > Uncomplicated Firewall.
 > Frontend for iptables aiming to make configuration of a firewall easier.
+> More information: <https://wiki.ubuntu.com/UncomplicatedFirewall>.
```
Comment (navarroaxel) [inline pages/linux/ufw.md:5]: Why not <https://manned.org/ufw> @bl-ue?

Comment (bl-ue) [reply pages/linux/ufw.md:5]: I didn't check either that or the wiki link. 😞

I tihnk the manned.org page is really useful tho, so I guess we should open a new PR to fix it 😂

**Comment by CleanMachine1** [comment]
Locking so comments aren't made here and on the other PR

**Comment by bl-ue** [comment]
@CleanMachine1 let's only do that when we have to — it's not critical to restrict conversation to a specific PR.

**Comment by CleanMachine1** [comment]
It stops confusion

**Pull request merged**